From 03348e9bec44e034f2aa942a18a55fb9f66960ac Mon Sep 17 00:00:00 2001
From: Liang Ma <liangma@bytedance.com>
Date: Sun, 18 Jul 2021 11:29:16 +0100
Subject: [PATCH 25/34] net/mlx5: export PMD-specific API file

[ upstream commit da7a5c14062ef48e2cbec1959e6fe89975498cfd ]

The file rte_pmd_mlx5.h should be exported by Meson.

Fixes: efa79e68c8cd ("net/mlx5: support fine grain dynamic flag")
Fixes: 23f627e0ed28 ("net/mlx5: add flow sync API")

Signed-off-by: Liang Ma <liangma@bytedance.com>
---
 drivers/net/mlx5/meson.build | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/net/mlx5/meson.build b/drivers/net/mlx5/meson.build
index e7495a764a..701afeab19 100644
--- a/drivers/net/mlx5/meson.build
+++ b/drivers/net/mlx5/meson.build
@@ -9,6 +9,7 @@ if not (is_linux or is_windows)
 endif
 
 deps += ['hash', 'common_mlx5']
+headers = files('rte_pmd_mlx5.h')
 sources = files(
 	'mlx5.c',
 	'mlx5_ethdev.c',
-- 
2.20.1

